### ISSUE TITLE
Author name link is sometimes null

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#3555]
See https://addons.mozilla.org/en-US/firefox/addon/adblock-57/

The developer link is https://addons.mozilla.org/en-US/firefox/addon/adblock-57/null

<img width="582" alt="add-ons_for_firefox" src="https://user-images.githubusercontent.com/1514/31734313-4de6da74-b40c-11e7-8a3d-6c7dd9a1f7ab.png">

**Comment by muffinresearch** [comment]
Looks like the url is null in API.  https://addons.mozilla.org/api/v3/addons/addon/adblock-57/

<img width="749" alt="nightly" src="https://user-images.githubusercontent.com/1514/31734566-17f04062-b40d-11e7-9d89-7e5badf822d3.png">

**Comment by muffinresearch** [comment]
I've filed https://github.com/mozilla/addons-server/issues/6726 re: the data being wrong - depending on the outcome of that will probably decide if we should do a quick fix for it or not?

**Comment by muffinresearch** [comment]
Looks like there's something that could be fixed on the API side, but until then it would be good to skip over a link that shows up as null in the data.

**Comment by vcarciu** [comment]
I reproduced this on -dev and URL is still null : https://addons-dev.allizom.org/api/v3/addons/addon/gaure-raill-dolphal/

**Comment by willdurand** [comment]
Looks good to me? See: https://addons-dev.allizom.org/en-US/firefox/addon/gaure-raill-dolphal/.

**Comment by willdurand** [comment]
There should not be any link when author url is null.

**Comment by vcarciu** [comment]
@willdurand : from what I understood from bug description I thought that it is not ok to show that url as null, but please correct me if I am wrong

**Comment by willdurand** [comment]
@vcarciu in the API you mean? This is a different issue then.

Here the issue is that `<a href="null">` urls are invalid. We should not display that. So author names should have a link when the API returns a link, and no link otherwise.

**Comment by vcarciu** [comment]
@willdurand : in that photo from @muffinresearch  : https://user-images.githubusercontent.com/1514/31734566-17f04062-b40d-11e7-9d89-7e5badf822d3.png ii se that the problem is that url is null and it is null for me too on -dev

**Comment by willdurand** [comment]
Yes, but this has been reported on the API https://github.com/mozilla/addons-server/issues/6726. Forget about this screenshot and only look at the one in the issue  :wink:

**Comment by vcarciu** [comment]
@willdurand , thanks for explanations marking issue as fixed based on https://addons-dev.allizom.org/en-US/firefox/addon/gaure-raill-dolphal/ which is working as expected
